.github/ISSUE_TEMPLATE/deprecation_template.md

```diff
@@ -0,0 +1,19 @@
+# DEPRECATION 
+
+**Feature:**
+
+**function:**  *filename.c*
+ 
+**bin/lib:** */directory*
+
+## Reason
+
+*Why should this feature be deprecated?  Possible reasons include being outside of RST's current scope or being a legacy function that is no longer used.*
+
+## Replacement
+
+*Would anything replace this feature?*
+
```

Comment:
```suggestion
## Backward compatibility

*Does this potentially influence any old file formats or old code? Does it change backward compatibility?*
```
Just a suggestion - maybe change the wording but it might be worth considering

Reply:
``` suggestion 
## Potential Issues 

- [ ] Backwards compatibility - *Does this potentially influence any old file formats or old code? Does it change backward compatibility?*
- [ ] reproducibility  - *Does this affect reproducibility between various RST versions?*

*If this affects one or both of the possible issues how would you inform or ensure people are aware and why this is needed.*
```

How about this? You make a good point and I realized the reproducibility issue too.